test/e2e/persistent_volumes.go

```diff
@@ -708,19 +850,19 @@ func makeWritePod(ns string, pvcName string) *api.Pod {
 			APIVersion: registered.GroupOrDie(api.GroupName).GroupVersion.String(),
 		},
 		ObjectMeta: api.ObjectMeta{
-			GenerateName: "write-pod-",
+			GenerateName: "client-",
 			Namespace:    ns,
 		},
 		Spec: api.PodSpec{
 			Containers: []api.Container{
 				{
 					Name:    "write-pod",
 					Image:   "gcr.io/google_containers/busybox:1.24",
-					Command: []string{"/bin/sh"},
-					Args:    []string{"-c", "touch /mnt/SUCCESS && (id -G | grep -E '\\b777\\b')"},
+					Command: []string{"/bin/sh", "-c"},
```

Comment:
There is no strict rule where '-c' really belongs, to me `Command` is the executable (bash) and `Args` are its arguments. Anyway, it's just a nit.